src/main/java/com/bathanh/exercise/basic/array/Ex207.java

```diff
@@ -0,0 +1,34 @@
+package com.bathanh.exercise.basic.array;
+
+public class Ex207 {
+
+    public int[] sortArrayAsc(final int[] array, final int a) {
+        int[] newArray = new int[array.length + 1];
+        int index = indexArray(array, a);
+        int j = 0;
+        for (int i = 0; i <= newArray.length - 1; i++) {
+            if (i == index) {
+                newArray[index] = a;
+            } else {
+                newArray[i] = array[j++];
+            }
+        }
+        return newArray;
+    }
+
+    public int indexArray(final int[] array, final int a) {
+        int low = 0, high = array.length - 1;
+        while (low <= high) {
+            int m = low + (high - low) / 2;
```

Comment:
Dòng này copy từ đâu đó đúng không 

```suggestion
            final int middle = low + (high - low) / 2;
```

Reply:
resolved